src/ui/components/Rating/index.js

```diff
@@ -46,6 +46,16 @@ export class RatingBase extends React.Component {
     this.props.onSelectRating(rating);
   }
 
+  checkStarRating = (rating, compareRating) => {
+    const roundRating = parseFloat(rating.toFixed(1));
```

Comment:
I think this might've been related to https://github.com/mozilla/addons-server/issues/7822 which is now fixed. We should't need to run `parseFloat` on this, and also if it's *not* a float I'm not sure `to.Fixed(1)` will work 😄